hazelcast/src/test/java/com/hazelcast/client/map/ClientMapStoreTest.java

```diff
@@ -249,6 +251,27 @@ public void destroyMap_configuredWithMapStore() {
         map.destroy();
     }
 
+    @Test(expected = NullPointerException.class)
```

Comment:
I'd consider replacing this to `ExpectedException` rule or to `try-catch` to be more concrete about where we expect the NPE and also to prevent false-positives.

Reply:
addressed via ExpectedException rule.